web/src/title/loader.ts

```diff
@@ -9,17 +9,44 @@ import { routeTool } from "../common/draw_area/stores";
 import {
   assetUrl,
   backend,
+  currentProjectKey,
   map,
   mode,
   one_destination,
-  projectName,
   route_pt_a,
   route_pt_b,
 } from "../stores";
 import { Backend } from "../wasm";
 
+// Returns whether the project name is already taken, otherwise the project is created.
```

Comment:
I see, this is only used for CNT right now, but we could cut over to it in NewProjectMode too.